### ISSUE TITLE
[Proposal/Idea] The Reward Chest (A Wood Burning Mechanism)

### ISSUE THREAD

**Issue by SFL-Aeon** [sunflower-land/sunflower-land#970]
The Problem: Currently there is an abundance of wood in the game with all the additional means to gain wood and far less means of burning the wood.

The Proposal: Create a daily chest (think similar to v1 chest) that players will “craft”. 

But what do you mean by craft? Didn’t you say this was similar to the v1 chest which we just logged in and claim after three days? Well yes and no. This will be gated by time (once a day) but is a means to burn wood not add an extra means of free income. We have a duty to look after the economy after all!

Ok, say we agree to crafting it instead of claiming once a day what kind of rewards are we talking about?

Fair enough, we can’t just ask you to craft something for the sake of crafting so here is the meat and potatoes of the proposal!

Once a day, each player has the choice to craft a chest which I am calling The Reward Chest.

Unlike before this chest will offer 3 options to craft: The Reward Chest (I), The Reward Chest (II), The Reward Chest (III).

The difference between the chest will be the amount of wood needed to craft them. 

My idea is that the T1 chest will take (10) wood to craft which is effectively 3 SFL with average luck chopping trees.

The T2 would take (100) wood or effectively 25 SFL.

The T3 would take (1000) wood effectively impossible for an average player but almost close to once a day for those with T3 beaver which is the main culprit for all that deforestation!

The idea would be to award 3 separate groups a chance to take a “risk” by crafting this chest for rewards as well as burn wood every day.

The Breakdown on Rewards: The idea is to create a sink of resources while at the same time reward players who take the “risk”.

T1 chest (10 wood aka 3 SFL): This chest would give a chance to award (1-5 SFL) to sweeten the pot I though of adding T1 crops to the loot table and possibly exp if enough are on board/devs agree.

T1 Crops: Sunflowers, potatoes, pumpkins and carrots.

There would be a 30% chance to get 1 SFL, 25% chance to get 2 SFL a 20% chance to get 3 SFL a 15% chance to get 4 SFL and a 10% chance to get 5 SFL.

On top of this you have 4 chances to either get 0-4 of the T1 crops. Such as 40% chance to get sunflowers (3 tiers: 100, 150, 300) 30% to get potatoes (25, 50, 100) 20% chance to get pumpkins (10, 20, 40) and 10% chance to get carrots (5, 10, 20).

If by some absolute stroke of luck you get the maximum rolls for loot you would gain 5 SFL + 300 sunflowers, 100 potatoes, 40 pumpkins and 20 carrots once a day. This equals 6.3 SFL if sold or roughly 2x your investment. (Just a side note the crops can be used for crafting so it’s not all about profits)

T2 Reward chest: (100 wood aka 25 SFL). This chest will award up to 10-40 SFL as well as T2 crops. (Yes the number is smaller than T1 it’s by design, the idea is a bigger risk for bigger rewards.)

T2 crops: Cabbage, Beetroot, Cauliflower 

There will be a 30% chance to get 10 SFL, a 25% chance to get 15 SFL, a 20% chance to get 20 SFL, a 15% chance to get 30 SFL and a 10% chance for 40 SFL.

Similar to T1 chest you have a chance to get 0-3 of the T2 crops.

45% chance for cabbage (50, 100, 150), 40% chance for beetroot (25, 50, 100) and 15% for cauliflower (25, 50, 75).

Similar to before, the best luck and roles would give 40SFL + 150 cabbage, 100 beetroots, 75 cauliflower or a total of 60.5 SFL or a minimum of 10 SFL no crops.

The final Tier is the trickiest because it needs to be rewarding enough to try yet not the new meta for SFL and throw the economy out of balance. So these numbers are all arbitrary and open for discussion!

T3 chest (1000 wood value ???) offer a chance to go big or leave empty handed! (You won’t my proposal will give something alway)

As usual New tier new tier crops!

T3 crops: Parsnip, Radish, Wheat.

The T3 is different as it will take the rewards of all chest and combine them along with adding a new elite reward for attempting this challenge!

The breakdown: 15% chance to get 5 SFL, 25% to get 10 SFL, 20% to get 20 SFL, 20% to get 30 SFL 15% to get 50 SFL, 4% to get 75 SFL and 1% to get 100 SFL!

Ontop of this you have a chance to roll all Tier crops (Lower tier have a higher chance of being awarded)

T1: 100% to get sunflowers with a high chance to get other T1.

100% sunflowers (1000)
50% Potatoes (1, 10, 50, 500)
30% Pumpkins (1, 5, 25, 250)
20% Carrots (1, 5, 25, 150)

T2: can be gained with T1 crops as well (separate table roll) moderate chance to roll for T2 crops but lower chance to gain

30% Cabbage (1, 10, 100, 200)
20% Beetroot (1, 10, 50, 100)
10% Cauli (1, 10, 25, 75)

T3: a small chance to roll this table yet no “fake rewards” if hit

15% Parsnip (5, 10, 25)
10% Radish ( 10, 25, 50)
5% Wheat (25, 50, 100)

A breakdown of you hit the lottery, 100 SFL + 1000 sunflowers, 500 potatoes, 250 pumpkins, 150 carrots, 200 cabbage, 100 beetroot, 75 cauli, 25 parsnip, 50 radish 100 wheat!

The jackpot total is 100 SFL + 63.656 SFL in crops.

consolidation: 5 SFL + sunflowers (1000)

While the T3 would technically cost you 250 axes the reward is aimed at T3 beaver holders and those high risk takers to burn wood supply for big rewards.

The TL;DR: this proposal is for burning wood yet giving all players a chance at rewards daily for a sense of some fun risk taking.

**Comment by GoblinNoLikePotato** [comment]
this is actually a brilliant solution for 'wood problems', I think a little bit of RNG with rewards is fun, and players can decide the 'risks' they want to take. very nice sink for the game and a little 'fun' everyday for everyone to enjoy. Definitely a good to have feature 👍

**Comment by SFL-Aeon** [comment]
Thanks! The idea was everyone could partake in it daily so it’s not focused on just one group. I think all people could play it once a day and it wouldn’t impact their gaines to much. And if halvening happens if axes don’t get cheaper we could always up the rewards slightly to keep it enticing!

**Comment by brian-man** [comment]
So the "wood problem" becomes an "SFL problem and all the other crops"?

**Comment by brian-man** [comment]
> T3 beaver which is the main culprit for all that deforestation!
Data?

**Comment by SFL-Aeon** [comment]
> So the "wood problem" becomes an "SFL and all the crops" problem?
> 
> Meaning the chest is effectively an "inconsistent dex" for converting wood to crops and SFL?
> 
> > T3 beaver which is the main culprit for all that deforestation!
> 
> Data?
> 
> Also, I thought Adam and/or Steve said part of Land Expansion was significant consumption of wood.

The idea was to add a means of something extra to chase after at the end of the day a simpler version would be to just make it similar to reward chest from v1 except require wood. Currently the SFL Gaines are being groped about so this effectively kills two birds with one stone.

I can go pull inuBakaBo excel sheet with the amount of wood being gained but yea T3 beaver is a big culprit for the increase in wood gaines

**Comment by SFL-Aeon** [comment]
On May 10th we had 2.78M wood which was before the beaver minting. After on May 11th this number dropped to 1.897M so almost 1M wood. Give that there was roughly 80-90 beavers left let’s say another 45K wood for 1.852M on May 11th.

Since then on May 19th we are sitting at 2.532M wood so almost in a week we gained 1M wood which took months and the migrations from v1 to get to. The wood is definitely an issue and thus the proposal XD

**Comment by Saipax** [comment]
I would honestly do it more like NFT winning style. For instance add 5 cosmetic NFTs with big supply and 1-2 NFTs with boosts to the chest.  The chest would be like 100-1000 wood.

It would help with burning wood without creating the problem with SFL inflation.

**Comment by SFL-Aeon** [comment]
> I would honestly do it more like NFT winning style. For instance add 5 cosmetic NFTs with big supply and 1-2 NFTs with boosts to the chest. The chest would be like 100-1000 wood.
> 
> It would help with burning wood without creating the problem with SFL inflation.

That is fair. This was something to give a slight change so people could give a risk aspect to it with a continuous means to burn wood.

**Comment by NoisyBucheron** [comment]
Just for info a T3 Beaver gives you a average of 500 wood per day **if you chop 24 times a day (impossible if you don't use a bot or a auto-click)**. 

An average T3 player chops half of that > 250 . So your data is not good.
The total population of T3 beavers chops per day 75,000 wood.

The real reasons for this wood inflation are:

-  the axes crafted to get SFL out of the game before token launch that still in circulation, 
-  the price of SFL right now, 
-  the lack of NFT's with wood requirement

Thank you

**Comment by SFL-Aeon** [comment]
> Just for info a T3 Beaver gives you a max of 576 wood per day **if you chop 24 times a day (impossible if you don't use a bot or a auto-click)**.
> 
> An average T3 player chops half of that > 288 . So your data is not good. The total population of T3 beavers chops per day 86,400 wood if done right with that average.
> 
> The real reasons for this wood inflation are:
> 
> * the axes crafted to get SFL out of the game before token launch that still in circulation,
> * the price of SFL right now,
> * the lack of NFT's with wood requirement
> 
> Thank you

While an average player could make this, this is definitely not the max. With every buff in game you could get up to 65% increase in wood. 

Granted I hear what you are saying. The 1k wood might be a stretch for most. The idea was it not to be done everyday. 

Without the T3 beaver, the wood would be a lot less. It is mainly the sole driving force of wood and on average accounts for 80% of the 110K wood introduced into the economy.

this proposal was to give a burn mechanism as well as a small means to gamble for SFL

**Comment by NoisyBucheron** [comment]
> 

The only buffs i'm not taking in consideration are the level 5 skill and the Discord mod buff, that's a 45% increase.
If you increase 45% more you get the 520>560 but if you're wake up 24H a day, if you see someone doing that it's almost sure they use auto-click.

**Comment by SFL-Aeon** [comment]
That’s fair, the proposal takes into account all buffs that exist in game. It also gives room for future buffs that may not exist as this will not be implemented over night. Some have suggest the average of 500 wood which can also be fine as well. 

I went with what the numbers add up to at a maximum efficient albeit questionable harvesting.

also the idea was for the T3 to be rarer than the t2 chest yet players can save up for T3 by doing a few T2 and taking a risk on T3 or giving people a means to purchase wood from others via trading to flesh out stored wood from others.
